qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
shouldn't this be done in the corresponding test suite, so the expectation can be controlled?